### ISSUE TITLE
Can't exclude file in directory

### ISSUE THREAD

**Issue by hoschi** [istanbuljs/nyc#296]
I have a file in `src/userAuth/userAuthReducer.js` which I can't exlude with its parent dir in the pattern.

this works
* **/userAuthReducer.js

this doesn't
* **/userAuth/userAuthReducer.js
* */userAuth/userAuthReducer.js
* src/userAuth/userAuthReducer.js

nyc version: 6.6.1
node version: 6.2.1
I use `babel-plugin-__coverage__` for instrumenting

**Comment by bcoe** [comment]
@hoschi this works fine for me:

```json
  "nyc": {
    "include": [
      "src/**/*.js"
    ],
    "exclude": [
      "src/foo/snuh.js"
    ],
    "require": [
      "babel-core/register"
    ],
    "sourceMap": false,
    "instrument": false
  },
```

I was having a bit of trouble but the underlying issue turned out to be the babel cache interfering with instrumentation, so I ran tests using:

`BABEL_DISABLE_CACHE=1 npm t`

try this and I think you'll find it does the trick.

**Comment by jamestalmage** [comment]
`nyc` should probably just disable the cache.

**Comment by skeep** [comment]
`BABEL_DISABLE_CACHE=1 npm t` works for me too. Looks like it's getting fixed in #303

**Comment by hoschi** [comment]
After upgrading nyc to 7.0.0 and switching to babel-plugin-istanbul this works now :)
